### ISSUE TITLE
OpenAPI 3.1

### ISSUE THREAD

**Issue by braco** [Mermade/widdershins#402]
Hi, do you have any plans to support OpenAPI 3.1?

https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0-rc1

**Comment by MikeRalphson** [comment]
Yes. If people have features of OpenAPI 3.1 they particularly want supporting as a priority, please list them here.

**Comment by MikeRalphson** [comment]
OpenAPI 3.1 GA has been released. You can track support for OAS 3.1 in the underlying library used by this project [here](https://github.com/Mermade/oas-kit/projects/1)

**Comment by jwillmer** [comment]
I'm using OpenAPI 3.1 and my generated API looks like this. I havent used `mermade` before so I don't know if this is a OpenAPI 3.1 issue or a config problem but my API title looks ugly.

![image](https://user-images.githubusercontent.com/1503577/140759841-657fa660-f974-401d-bf99-8eecc72d24a5.png)

**Comment by MikeRalphson** [comment]
@jwillmer have you tried adding `operationId`s to your input OpenAPI definition?

**Comment by afilina** [comment]
I rely on `webhooks` a lot, and this is only available since OpenAPI 3.1. This means that webhooks are ignored when generating the markdown file. I only tried node for like a week more than 10 years ago, so I won't be of much help on the keyboard. Is there anything else I can do to help?
